db-client-java/src/main/java/com/eventstore/dbclient/EventStoreDBPersistentSubscriptionsClient.java

```diff
@@ -11,20 +11,52 @@ public static EventStoreDBPersistentSubscriptionsClient create(EventStoreDBClien
         return new EventStoreDBPersistentSubscriptionsClient(settings);
     }
 
+    /**
+     * @deprecated prefer {@link #createToStream(String, String)}
+     */
+    @Deprecated
     public CompletableFuture create(String stream, String group) {
-        return this.create(stream, group, CreatePersistentSubscriptionOptions.get());
+        return this.createToStream(stream, group);
     }
 
+    public CompletableFuture createToAll(String group) {
+        return this.createToAll(group, CreatePersistentSubscriptionToAllOptions.get());
+    }
+
+    public CompletableFuture createToStream(String stream, String group) {

```

Comment:
**CONCERN:** Based on the [RFC](https://github.com/EventStore/home/pull/550/files#diff-ad3a37ec8a5416ddcd905109248d13dba6a72a352ce7ee41b9d3d004aae1ff95R18), this method is redundant, as we can just use `create`. The same applies to other `ToStream` management methods.

Reply:
Removed `toStream` classes and methods.